BSD cp doesn't support "-d" to preserve symlinks, but "-R" does the same.
--- configure.orig	2005-01-02 15:19:23.000000000 -0600
+++ configure	2011-02-09 15:34:24.000000000 -0600
@@ -29,7 +29,7 @@
 LFS="yes"
 
 # cp flags
-cpflags="d"
+cpflags="R"
 
 # some stuff for MacOSX -----------------------------
 HOSTNAME=`./config.guess`
